### PR TITLE
FeatureForm: localized messages

### PR DESCRIPTION
**Pull request by dotMorten**
Adds better localizable error messages to form validation and handles null and invalid values in text boxes better.

### CODE REVIEW

File: src/Toolkit/Toolkit/UI/Controls/FeatureForm/FieldFormElementView.cs [hunk 1]
```diff
@@ -118,12 +118,12 @@ private async void OnValuePropertyChanged()
...
             {
                 await FeatureForm.EvaluateExpressionsAsync();
-                var errors = Element?.GetValidationErrors();
+                var errors = Element.GetValidationErrors();
 
                 string? errMessage = null;
                 if (errors != null && errors.Any())
                 {
-                    errMessage = string.Join("\n", errors.Select(e => e.Message));
+                    errMessage = string.Join("\n", errors.Select(e => FeatureFormView.ValidationErrorToLocalizedString(Element, e)!));
```
Comment (mstefarov) [inline src/Toolkit/Toolkit/UI/Controls/FeatureForm/FieldFormElementView.cs:126]: Should this be `Environment.NewLine`? Or does `\n` work everywhere?

File: src/Toolkit/Toolkit.WPF/UI/Controls/FeatureForm/FeatureFormView.Theme.xaml [hunk 2]
```diff
@@ -36,7 +36,7 @@
       <Setter.Value>
         <ControlTemplate TargetType="{x:Type primitives:DateTimePickerFormInputView}">
           <StackPanel>
-            <DatePicker x:Name="DatePicker" SelectedDate="{Binding Value, Mode=OneWay}" IsEnabled="{TemplateBinding IsEnabled}" />
+            <DatePicker x:Name="DatePicker" IsEnabled="{TemplateBinding IsEnabled}" />
```
Comment (mstefarov) [inline src/Toolkit/Toolkit.WPF/UI/Controls/FeatureForm/FeatureFormView.Theme.xaml:39]: Why this change? Does DatePicker still initialize correctly?

File: src/Toolkit/Toolkit/UI/Controls/FeatureForm/FieldFormElementView.cs (hunk 1 above)
Comment (dotMorten) [reply src/Toolkit/Toolkit/UI/Controls/FeatureForm/FieldFormElementView.cs:126]: Works everywhere.

File: src/Toolkit/Toolkit.WPF/UI/Controls/FeatureForm/FeatureFormView.Theme.xaml (hunk 2 above)
Comment (dotMorten) [reply src/Toolkit/Toolkit.WPF/UI/Controls/FeatureForm/FeatureFormView.Theme.xaml:39]: The control itself handles updating this - with the binding two things were updating, causing one edit to overwrite the other, preventing you from setting a date to null.

**Review by mstefarov — APPROVED** [review]
